Add privacy policy and terms of service links to register page

arslanashraf7 opened the pull request:
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
closes #197 

#### What's this PR do?
- Adds a privacy policy and terms of service links on the `Create Account` page

#### How should this be manually tested?
- Visit `/create-account/` and verify that you can see the privacy policy and terms of service links there.

#### Screenshots (if appropriate)

**Desktop Normal Mode**
<img width="696" alt="Screenshot 2021-09-16 at 7 13 20 PM" src="https://user-images.githubusercontent.com/34372316/133628409-62dae17c-2d1b-4a37-9e94-0c1663112f92.png">

**Mobile Normal Mode**
<img width="311" alt="Screenshot 2021-09-16 at 7 14 10 PM" src="https://user-images.githubusercontent.com/34372316/133628478-c8d25743-8990-4679-a8c4-e2a7fb62822b.png">

**Desktop Error Mode**
<img width="698" alt="Screenshot 2021-09-16 at 7 13 35 PM" src="https://user-images.githubusercontent.com/34372316/133628601-af832b9b-7704-4001-928c-992e6c86229a.png">

**Mobile Error Mode**
<img width="314" alt="Screenshot 2021-09-16 at 7 13 51 PM" src="https://user-images.githubusercontent.com/34372316/133628684-4dbf46a7-5742-4f6e-9086-40f7bdbaea01.png">

pdpinch commented on static/scss/auth.scss:
```diff
@@ -242,3 +242,8 @@
 }
+
+.policy-consent {
+  font-size: 14px;
```
I don't think this needs to be smaller than the body text.

arslanashraf7 replied:
Resolved.

pdpinch reviewed (commented):
Screenshots are fine. I left a small comment about font size, but it's not a big deal & we can leave as is in the interests of time.

arslanashraf7 commented:
> Screenshots are fine. I left a small comment about font size, but it's not a big deal & we can leave as is in the interests of time.

@pdpinch, I've addressed your comment and removed the custom font styling. The screenshots have also been updated in the PR description for your review. Let me know if you'd want any changes to them.

asadiqbal08 commented on static/js/components/forms/RegisterEmailForm.js:
```diff
@@ -45,6 +46,15 @@ const RegisterEmailForm = ({ onSubmit }: Props) => (
           <ErrorMessage name="email" component={FormError} />
+          <p className="py-2">
+            By creating an account I agree to the{" "}
+            <a href={routes.informationLinks.termsOfService}>
```
Should the link not be opened in a new tab ?

arslanashraf7 replied:
I thought the user doesn't need to go to a new tab he needs to read `Policy links` because they are going to be a decisive factor for the user at that point and we should keep the same tab for it and let them come back to register.

Additionally, I also checked the policy links with the footer and they also open in the same tab so the behavior would also be standardized if I keep this behavior.

Does this answer your concern @asadiqbal08 ?

FYI, @pdpinch

asadiqbal08 reviewed (approved):
LGTM 👍 , looking for an answer to a question just.

pdpinch commented:
Thanks for the question @asadiqbal08 -- I think the privacy policy and terms of service should open in a new tab, to avoid user anxiety about losing the data they put into the form. Even if the data isn't lost, many users think it will be lost. 

This doesn't apply to the links in the footer.
